### PR TITLE
have topomap use _time_mask

### LINKED ISSUE #2901: plot_evoked_topomap use _time_mask
When figuring out which time points to plot, a straight comparison 
```
evoked.times >= t_desired
```
is used. This is not rounding-safe: if `t_desired` happens to be the last sample in `evoked.times` AND there's rounding error making the result an empty list.

Fix (now actually working) proposed in #2895

### PR DESCRIPTION
**Pull request by cjayb**
@Eric89GXL try again...

floating point rounding-safe way of getting time points to plot in `plot_topomap`

closes #2901

### CODE REVIEW

**Comment by jaeilepp** [comment]
Is this fixing some issue? Many of the tests seem to fail.
You might want to rebase now: https://github.com/mne-tools/mne-python/pull/2894

**Comment by cjayb** [comment]
Sorry, will create an issue asap to explain, think I may have made a brain dead mistake (although the need for a change is probably real)

**Comment by jaeilepp** [comment]
LGTM

File: mne/viz/topomap.py [hunk 1]
```diff
@@ -1168,7 +1168,9 @@ def plot_evoked_topomap(evoked, times="auto", ch_type=None, layout=None,
     fig = axes[0].get_figure()
     fig.subplots_adjust(left=w_frame, right=1 - w_frame, bottom=0,
                         top=1 - top_frame)
-    time_idx = [np.where(evoked.times >= t)[0][0] for t in times]
+    # find first index that's >= (to rounding error) to each time point
+    time_idx = [np.where(_time_mask(evoked.times, t, None))[0][0]
```
Comment (dengemann) [inline mne/viz/topomap.py]: you could actually pass sfreq too

Comment (jaeilepp) [inline mne/viz/topomap.py]: Yeah, but in my opinion, this is already a bit 'heavy' for a corner case. This should work well enough without the sfreq.

Comment (dengemann) [inline mne/viz/topomap.py]: what would we loose by passing it?

Comment (jaeilepp) [inline mne/viz/topomap.py]: Few rounds of computations :)
But this gets called max 20 times per plot, so I'm fine with either way.

Comment (dengemann) [inline mne/viz/topomap.py]: if this is a concern we should rewrite matplotlib and its backends in
cython or so ...

**Comment by cjayb** [comment]
prefer not to work on mpl backends ;)

**Comment by Eric89GXL** [comment]
+1 for merge from me. @jaeilepp okay with you?

**Comment by jaeilepp** [comment]
Yes. LGTM. Merging.

**Comment by jaeilepp** [comment]
thx @cjayb

**Pull request merged**